micromanager_gui/_util.py

```diff
@@ -170,3 +173,19 @@ def __init__(
 
     def currentText(self) -> str:
         return self._combo.currentText()
+
+
+def set_wdg_color(color: str, wdg: QWidget):
+    wdg.setStyleSheet(f"color: {color};")
+
+
+def get_dev_prop_val(
+    group: str, preset: str, mmcore: Optional[CMMCorePlus] = None
+) -> list:
+    mmc = mmcore or get_core_singleton()
+    return [(k[0], k[1], k[2]) for k in mmc.getConfigData(group, preset)]
+
+
+def get_dev_prop(group: str, preset: str, mmcore: Optional[CMMCorePlus] = None) -> list:
+    mmc = mmcore or get_core_singleton()
+    return [(k[0], k[1]) for k in mmc.getConfigData(group, preset)]
```

Comment:
these need docstrings explaining what they do, looking at them quickly I'm not totally sure I understand.. You can feel free to assume that the reader is knowledgeable about the internals of napari-micro when writing them.